templates/checks.json

```diff
@@ -0,0 +1,11 @@
+{
+  "checks": { {% for check in checks %}
+    "{{check.type}}": {
+      "command": "{{check.script}}",
+      "standalone": true,
+      "interval": 5,
+      "type": "metric",
+      "subscribers": ["{{check.subscribers}}"]
+    }{%- if not loop.last %},{% endif %}{% endfor %}
```

Comment:
This is needed, if a space is placed, Sensu sees this as an invalid json if I remember correctly. 
It would be possible, but I prefer using templates for services, since it provides cleaner separation of charm logic and service logic, plus leads to easier understanding a services needed config options